src/Frontend/Testing.hs

```diff
@@ -0,0 +1,19 @@
+{-# OPTIONS_GHC -Werror -Wall #-}
+
+module Frontend.Testing
+where
+
+import Control.Lens ((^.))
+
+import Action
+import Persistent
+import Types
+
+-- | Make a topic timeout if the timeout is applicable for the topic.
+-- FIXME: Make timeout the jury phase too.
+makeTopicTimeout :: (ActionPersist r m, ActionUserHandler m) => AUID Topic -> m ()
+makeTopicTimeout tid = do
+    Just topic <- persistent $ findTopic tid -- FIXME: 404
```

Comment:
this 404 will be thrown by `PersistM`, and we don't have to worry about it here.